### PR TITLE
Removal of financial date when displaying available transfer funds

### PR DESCRIPTION
**Pull request by ogrwill**

### CODE REVIEW

File: src/SFA.DAS.LevyTransferMatching.Web/Views/Opportunities/Index.cshtml [hunk 1]
```diff
@@ -67,7 +67,7 @@
                                     <h2 class="govuk-heading-m das-search-results__heading govuk-!-margin-bottom-0">
                                         <a href="@Url.Action("Detail", new { EncodedPledgeId = opportunity.ReferenceNumber })" class="das-search-results__link">@opportunity.EmployerName (@opportunity.ReferenceNumber)</a>
                                     </h2>
-                                    <p class="govuk-hint">up to @opportunity.DisplayAmount available in @Model.TaxYear</p>
```
Comment (chrisfoster76) [inline src/SFA.DAS.LevyTransferMatching.Web/Views/Opportunities/Index.cshtml:70]: I'm guessing that the removal of this property reference in the markup leaves it unused, in which case we can remove it from the model as well.

File: src/SFA.DAS.LevyTransferMatching.Web/Views/Shared/_OpportunitySummary.cshtml [hunk 2]
```diff
@@ -23,7 +23,7 @@
     @if (!Model.HideFooter)
    {
         <div class="app-highlight__footer">
-            <p class="govuk-heading-s">Total funding available: up to £@Model.Amount.ToString("N0") available in @Model.YearDescription</p>
+            <p class="govuk-heading-s">Total funding available: £@Model.Amount.ToString("N0")</p>
```
Comment (chrisfoster76) [inline src/SFA.DAS.LevyTransferMatching.Web/Views/Shared/_OpportunitySummary.cshtml:26]: As above, if this property on the model is no longer used, it should be removed from the model, the mapper and any unit tests.

**Review by chrisfoster76 — CHANGES_REQUESTED** [review]
Just a request to remove the properties from the models as well.

File: src/SFA.DAS.LevyTransferMatching.Web/Views/Opportunities/Index.cshtml (hunk 1 above)
Comment (ogrwill) [reply src/SFA.DAS.LevyTransferMatching.Web/Views/Opportunities/Index.cshtml:70]: Done

**Pull request merged**